File: pset1/cash/cash.c
#include<stdio.h>
#include<cs50.h>
#include<math.h>

int main(void)
{
    float change;
    int n = 0;
    do
    {
     change = get_float("Change owed:");
    }
    while(change < 0);
    change = round(change * 100);
    while(change >= 25)
        {
            change = change - 25;
            n++;
        }
    while(change >= 10)
        {
            change = change - 10;
            n++;
        }
    while(change >= 5)
        {
            change = change - 5;
            n++;
        }
    while(change >= 1)
        {
            change = change - 1;
            n++;
        }
    
    
    
    
    printf("%d\n",n);
}

File: pset6/cash/cash.py
from cs50 import get_float
import sys
while True:
    change = get_float("Change owed: ")
    if change > 0:
        break
change = round(change * 100)

count = 0
while change > 0:
    if change >=25:
        change = change - 25
        count = count + 1
    elif change >=10:
        change = change - 10
        count = count + 1    
    elif change >=5:
        change = change - 5
        count = count + 1        
    elif change >=1:
        change = change - 1
        count = count + 1    
    
print(count)        